workflow/util/util.go

```diff
@@ -155,14 +156,17 @@ func SubmitWorkflow(wfIf v1alpha1.WorkflowInterface, wf *wfv1.Workflow, opts *Su
 	if opts.ServiceAccount != "" {
 		wf.Spec.ServiceAccountName = opts.ServiceAccount
 	}
+	labels := wf.GetLabels()
+	if opts.Labels != "" {
+		labels, _ = cmdutil.ParseLabels(opts.Labels)
```

Comment:
Can we silently ignore an error from this method?

Reply:
Labels from a workflow are completely ignored if labels are set by the command line flag. I think we should merge the labels.